### PR TITLE
libmisc/root_flag: add tips for --root flag only support absolute path

### PR DESCRIPTION
**Pull request by CoelacanthusHex**
- Add tips in error message.
- Add tips in man.
- Add zh_CN and zh_TW for tips.

Signed-off-by: Celeste Liu <coelacanthus@outlook.com>

### CODE REVIEW

**Comment by CoelacanthusHex** [comment]
Look like failed CI is caused by not finding the ubuntu impish repo rather than my changes.

File: libmisc/root_flag.c [hunk 1]
```diff
@@ -79,7 +79,7 @@ static void change_root (const char* newroot)
 
 	if ('/' != newroot[0]) {
 		fprintf (log_get_logfd(),
-		         _("%s: invalid chroot path '%s'\n"),
+		         _("%s: invalid chroot path '%s', only support absolute path.\n"),
```
Comment (hallyn) [inline libmisc/root_flag.c:82]: Sorry, but could you change this to something like "only absolute paths are supported" or "an absolute path is required"?

Comment (CoelacanthusHex) [reply libmisc/root_flag.c]: > Sorry, but could you change this to something like "only absolute paths are supported" or "an absolute path is required"?

done

File: man/chage.1.xml [hunk 2]
```diff
@@ -194,6 +194,7 @@
 	    Apply changes in the <replaceable>CHROOT_DIR</replaceable>
 	    directory and use the configuration files from the
 	    <replaceable>CHROOT_DIR</replaceable> directory.
+	    Only support absolute path.
```
Comment (hallyn) [inline man/chage.1.xml:197]: I meant in the manpage too - "Only absolute paths are supported."  If you prefer I can change it inline as I apply.

Comment (CoelacanthusHex) [reply man/chage.1.xml]: I'm sorry for misunderstanding your meaning.
done

**Comment by hallyn** [comment]
Thank you.

**Pull request merged**